docs/configuration.md

```diff
@@ -1752,6 +1752,14 @@ showDF(properties, numRows = 200, truncate = FALSE)
     Executable for executing R scripts in client modes for driver. Ignored in cluster modes.
   </td>
 </tr>
+<tr>
+  <td><code>spark.r.shell.command</code></td>
+  <td>R</td>
+  <td>
+    Executable for executing sparkR shell in client modes for driver. Ignored in cluster modes. It is the same as environment variable <code>SPARKR_DRIVER_R</code>, but take precedence over it.
+    <code>spark.r.shell.command</code> is used for interactive mode of sparkR (sparkR shell) while <code>spark.r.driver.command</code> is used for the batch mode (running sparkR script).
```

Comment:
Hmm, I'm not entirely following.
`sparkR` is the name of the shell script that an user can run from commandline/their shell (lower case)
`SparkR` is how we could call R support of Spark (upper case)